index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="index.css">
+  <title>Awesome Books with ES6 Modules</title>
+</head>
+<body>
+  <header>
+    <nav>
+        <a class="logo" href="#">Awesome Books</a>
+        <div>
+            <ul class="links">
+                <li id="list"><a href="#">List</a></li>
+                <li id="add-new"><a href="#">Add new</a></li>
+                <li id="contact"><a href="#">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+    <div id="date"></div>
+  </header>
+  <section id="lists">
+    <h1>Awesome Books</h1>
+    <div class="books">
+      <ul id="book-lists"></ul>
+    </div>
+  </section>
+  <!--hr-->
+  <section id="forms">
+    <h2>Add a new book</h2>
+    <form class="form">
+      <input type="text" name="title" placeholder="title" id="title"><br>
+      <br>
+      <input type="text" name="author" placeholder="author" id="author"><br>
+      <br>
+      <button type="submit" id="button">Add</button>
+    </form>
+  </section>
```

Comment:
- [ ] 👍 As I previously mentioned, the navigation items are working perfectly, and it's great to see the "Add a new book" section displayed by default. However, based on the project requirement, when the page is refreshed or opened, it should show the "Awesome Books list" section by default instead. To align with this requirement, please consider updating your code and taking the necessary steps to ensure that the "Awesome Books list" section is displayed as the default section when the page is refreshed or opened. 🙏 

[Awesome-Books-with-ES6-Modules.webm](https://github.com/Omwomo/Awesome-books-with-ES6/assets/55840999/8a6182fa-4537-430b-a63f-3560e62a8060)

By making this adjustment, you will ensure that the project meets the specified functionality and provides a smoother user experience. Keep up the good work, and I'm confident you'll handle this update with ease! 🚀📚